.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
+package-lock.json
```

Comment:
nice catch :)

Reply:
in other projects we have `package-lock=false` in the npmrc file, but I think this is a better solution than that as developers now have the local file and can use `npm ci` to execute an install if they want that.